apps/web/playwright/organization/organization-invitation.e2e.ts

```diff
@@ -38,21 +42,24 @@ test.describe("Organization", () => {
         page.locator(`[data-testid="email-${invitedUserEmail.replace("@", "")}-pending"]`)
       ).toHaveCount(1);
 
-      // eslint-disable-next-line playwright/no-conditional-in-test
-      if (!inviteLink) return null;
+      assertInviteLink(inviteLink);
```

Comment:
Instead of not running the assertions, fail the test if inviteLink isn't available.